pages/1.11/installing-upgrading/cloud/gce/index.md

```diff
@@ -0,0 +1,285 @@
+---
```

Comment:
This page needs major revisions by the engineers that worked on it. If those merge before this pr does, make sure not to overwrite them.